docs/auto-instrumentation-install.md

```diff
@@ -235,20 +380,58 @@ provides best effort support with issues related to native OpenTelemetry instrum
 | python                  | Splunk        | Coming Soon |                  |                              | [Link](github.com/signalfx/splunk-otel-python)                                       |                                                                                |
 | java                    | OpenTelemetry | Available   | Yes              | Mostly                       | [Link](https://github.com/open-telemetry/opentelemetry-java-instrumentation)         | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java         |
 | dotnet                  | OpenTelemetry | Available   | Yes              | Mostly                       | [Link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)       | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet       |
-| nodejs                  | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-nodejs-instrumentation)       | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodes        |
+| nodejs                  | OpenTelemetry | Available   | Yes              | Mostly                       | [Link](https://github.com/open-telemetry/opentelemetry-nodejs-instrumentation)       | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodes        |
 | python                  | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-java-instrumentation)         | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java         |
 | apache-httpd            | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-apache-httpd-instrumentation) | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
 
-</details>
-
 ### Documentation Resources
 
 - https://developers.redhat.com/devnation/tech-talks/using-opentelemetry-on-kubernetes
 - https://github.com/open-telemetry/opentelemetry-operator/blob/main/README.md
 - https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation
 - https://github.com/open-telemetry/opentelemetry-operator/blob/main/README.md#opentelemetry-auto-instrumentation-injection
 - https://github.com/open-telemetry/opentelemetry-operator/blob/main/README.md#use-customized-or-vendor-instrumentation
+
+### Troubleshooting the Operator and Cert Manager
+
+#### 1. Check the logs for failures
+
+**Operator Logs:**
+
+```bash
+kubectl logs -l app.kubernetes.io/name=operator
+```
+
+**Cert-Manager Logs:**
+
+```bash
+kubectl logs -l app=certmanager
+kubectl logs -l app=cainjector
+kubectl logs -l app=webhook
+```
+
+#### 2. Cert-Manager Issues
```

Comment:
Can we also mention that user has the option of getting a self-signed certificate generated for the webhook using the chart? We don't seem to call out this option in the installation readme.

I understand cert-manager will provide full certificate lifecycle management including replacing expiring cert, but it is a relatively difficult application to operate and troubleshoot. We don't want a user trying out the operator to be blocked by a cert-manager issue when there is an alternative to use.

Reply:
Done